src/network/serverpackethandler.cpp

```diff
@@ -629,13 +629,19 @@ void Server::handleCommand_InventoryAction(NetworkPacket* pkt)
 		if (ma->from_inv != ma->to_inv)
 			m_inventory_mgr->setInventoryModified(ma->to_inv);
 
-		bool from_inv_is_current_player =
-			(ma->from_inv.type == InventoryLocation::PLAYER) &&
-			(ma->from_inv.name == player->getName());
-
-		bool to_inv_is_current_player =
-			(ma->to_inv.type == InventoryLocation::PLAYER) &&
-			(ma->to_inv.name == player->getName());
+		bool from_inv_is_current_player = ma->from_inv.type == InventoryLocation::CURRENT_PLAYER;
```

Comment:
`CURRENT_PLAYER` is only used for client-side applied changes and is ignored server-side. See serveinventorymgr.cpp L44 for example. Though it does not break anything, this check is superfluous.

Reply:
I see, I should've checked this. The line can be changed to `= false;` then